File: soal1.pas
program Soal1;
uses crt;
var
    bilangan: integer;
begin
    clrscr;
    write('Masukkan bilangan bulat positif: ');
    readln(bilangan);

    if bilangan mod 4 = 0 then
        writeln(bilangan, ' merupakan kelipatan 4')
    else
        writeln(bilangan, ' bukan kelipatan 4');

    readln;
end.

File: soal1.c
#include <stdio.h>

int main() {
    int bilangan;

    printf("Masukkan bilangan bulat positif: ");
    scanf("%d", &bilangan);

    if (bilangan % 4 == 0)
        printf("%d merupakan kelipatan 4\n", bilangan);
    else
        printf("%d bukan kelipatan 4\n", bilangan);

    return 0;
}